examples/Titanic/README.md

```diff
@@ -1,73 +1,70 @@
-## Using Captum to interpret Pytorch models and serving on Torchserve
-
-In this example, we will demonstrate the basic features of the [Captum](https://captum.ai/) interpretability,and serving the model on torchserve through an example model trained on the Titanic survival data. you can download the data from [titanic](https://biostat.app.vumc.org/wiki/pub/Main/DataSets/titanic3.csv)
-
-We will first train a deep neural network on the data using PyTorch and use Captum to understand which of the features were most important and how the network reached its prediction.
-
-you can get more details about used attributions methods used in this example
-
-1. [Titanic_Basic_Interpret](https://captum.ai/tutorials/Titanic_Basic_Interpret)
-2. [integrated-gradients](https://captum.ai/docs/algorithms#primary-attribution)
-3. [layer-attributions](https://captum.ai/docs/algorithms#layer-attribution)
- 

```

Comment:
@KasirajanA  Please do not remove these lines keep as it is.

Reply:
@Arvind-Ideas2IT , Added the requested lines